app/routes/reports.tsx

```diff
@@ -74,8 +74,6 @@ export default function Reports() {
 					</Link>
 				))}
 			</section>
-
-			<Outlet />
```

Comment:
i thought we wanted return the filtered list of reports here, with a nested route `/reports/:query-string`

maybe need to rename `/reports_.$slug.tsx` --> `/report..` or `/report-detail..`, so it's not a child of `/reports` ?

Reply:
My intention is not for queries to function as nested routes. The URL in my mind looks like: 

`/reports?q="water"&region="Abud"&sort="needed"`